portfolio/src/main/webapp/script.js

```diff
@@ -16,12 +16,51 @@
  * Cycles through greetings in different languages for front page. 
  */
 const greetings =['Hi, nice to meet you!', '¡Hola, encantado de conocerte!', '你好，很高興見到你', 'Salut! Enchanté!'];
-var textElement = document.getElementById('changeText');
+let textElement = document.getElementById('changeText');
```

Comment:
const

Reply:
I'm going to guess the choice of `let` was motivated by the fact that textElement is mutated on the next line. The distinction is subtle - const does not refer to the object in question being constant, but the _binding_ being constant e.g. the variable `textElement` always refers to this object and will never refer to anything else. The underlying object can still be mutated e.g. a const array can have elements added and removed from it.